packages/kit-bg/src/services/ServiceSwap.ts

```
@@ -1139,6 +1233,22 @@ export default class ServiceSwap extends ServiceBase {
     }
   }
 
+  @backgroundMethod()
+  async speedSwapApprovingStateAction() {
+    void this.cleanSpeedSwapApprovingInterval();
+    const approvingTransaction = await this.getSpeedSwapApprovingTransaction();
+    if (approvingTransaction && approvingTransaction.txId) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)